templates/bot/js/notification-function-base/README.md

```diff
@@ -1,104 +1,274 @@
 
-The core notification implementation is in `bot/` folder, containing following content:
+The following files are project-related files. You generally will not need to customize these files.
 
 | File / Folder | Contents |
 | - | - |
-| `messageHandler/` | The function to handle bot messages |
-| `*Trigger/` | The function to trigger notification |
-| `src/adaptiveCards/` | Adaptive card templates |
-| `src/internal/` | Generated initialize code for notification functionality |
-| `src/cardModels.*s` | Adaptive card data models |
-| `src/*Trigger.*s` | The entrypoint of each notification trigger |
-| `.funcignore` | The azure functions ignore file to exclude local files |
-| `.gitignore` | The git ignore file to exclude local files from bot project |
-| `host.json` | The azure functions host file |
-| `local.settings.json` | The azure functions local setting file |
-| `package.json` | The NPM package file for bot project |
+| `.funcignore` | Azure Functions ignore file to exclude local files |
+| `.gitignore` | Git ignore file |
+| `host.json` | Azure Functions host file |
+| `local.settings.json` | Azure Functions settings for local debugging |
+| `package.json` | NPM package file |
+
+# Customize your application
+
+By default:
+
+* If you selected `timer`, will send a notification to Teams every 30 seconds
+* If you selected `http trigger`, will provide a Azure Function http trigger that will send a notification to Teams in response
+
+This section outlines some customization you can do to adopt the application for your needs.
+
+## Customizing the event source
+
+If you selected `timer` trigger, the default Azure Function timer trigger (`src/timerTrigger.js`) implementation simply sends a hard-coded Adaptive Card every 30 seconds.
+
+If you selected `http` trigger, when this trigger is hit (via a HTTP request), the default implementation sends a hard-coded Adaptive Card to Teams.
+
+You can customize this behavior by customizing `src/*Trigger.js`. A typical implementation might make an API call to retrieve some events and/or data, and then send an Adaptive Card as appropriate.
+
+Teams Toolkit enables you to [easily connect to an existing API](#connect-to-existing-api).
+
+## Customize the Adaptive Card
+
+You can edit the file `src/adaptiveCards/notification-default.json` to customize the Adaptive Card to your liking. The file `src/cardModels.ts` defines a data structure that is used to fill data for the Adaptive Card.
+
+The binding between the model and the Adaptive Card is done by name matching (for example,`CardData.title` maps to `${title}` in the Adaptive Card). You can add, edit, or remove properties and their bindings to customize the Adaptive Card to your needs.
+
+You can also add new cards if appropriate for your application. Please follow this [sample](https://aka.ms/teamsfx-adaptive-card-sample) to see how to build different types of adaptive cards with dynamic contents.
```

Comment:
Better to mention here we have adaptive card samples for columnset, list, factset, mention to attract more people click the link.

Reply:
Updated.